neutronics/mcnp_inputs.py

```diff
@@ -31,20 +31,22 @@ class HomogeneousInput:
     """Write Homogeneous Input File.
     Class to write homogeneous MCNP burnup input files.
     """
-    
-    def __init__(self, radius, length, pnnl_mats, thick_refl=15):
+    kW_to_MW = 0.001    
+    def __init__(self, radius, length, power, pnnl_mats, thick_refl=15):
         """Initialize geometric reactor parameters.
 
         Initialized Attributes:
         -----------------------
             z (float): reactor core height
             r (float): reactor core radius
             frac_fuel (float): fuel to coolant channel fraction
+            Q_therm (float): reactor thermal power
```

Comment:
maybe it is just me, but it feel strange to call power `Q`... I am used to `P`

Reply:
P is often for pressure...

Reply:
Should definitely list units in the docstring if they are required.